src/Features/Core/Portable/Common/Glyph.cs

```diff
@@ -4,6 +4,8 @@ namespace Microsoft.CodeAnalysis
 {
     internal enum Glyph
     {
+        None,
+
```

Comment:
@mgoertz-msft @paulvanbrenk @jramsay .  After this change goes in, you'll want to recompile any binaries you have that depend on this internal enum.  Otherwise, you'll be referencing the wrong items.

For APIs you have that use the new 'Tag' based appraoch, there's nothing you'll need to do.